orchagent/portsorch.cpp

```diff
@@ -2873,3 +2873,49 @@ void PortsOrch::doTask(NotificationConsumer &consumer)
         sai_deserialize_free_port_oper_status_ntf(count, portoperstatus);
     }
 }
+
+/*
+ * sync up orchagent with libsai/ASIC for port state.
+ *
+ * Currently NotificationProducer is used by syncd to inform port state change,
+ * which means orchagent will miss the signal if it happens between orchagent shutdown and startup.
+ * Syncd doesn't know whether the signal has been lost or not.
+ * Also the source of notification event is from libsai/SDK.
+ *
+ * Latest oper status for each port is retrieved via SAI_PORT_ATTR_OPER_STATUS sai API,
+ * the hostif and db are updated accordingly.
+ */
+void PortsOrch::syncUpPortState()
```

Comment:
>syncUpPortState [](start = 16, length = 15)

syncUpPortStatus

Reply:
For warm start, we have restore phase and sync up phase, that is one of reasons the name of syncUpPortState() is used.   My focus is on the function part.  It looks you have strong opinion on the name,  I could change it to  refreshPortStatus().